### ISSUE TITLE
[a11y] 11.1.4.11 Non-text contrast

### ISSUE THREAD

**Issue by tschiebel** [owncloud/ios-app#1335]
audit: https://infinite.owncloud.com/s/qSvYucPltEYUhhc

https://github.com/BIK-BITV/BIK-App-Test/tree/main/Pr%C3%BCfschritte/de
https://www.etsi.org/deliver/etsi_en/301500_301599/301549/03.02.01_60/en_301549v030201p.pdf

Note: Information-bearing graphics and graphic control elements as well as their states should
states should have a contrast of at least 3:1 to adjacent colors.

### Personal Space

- ~~[ ] Each directory or file in the list has "status" icons,
which indicate to users whether the file has already been shared, for example.
Unfortunately, some of these icons do not meet the contrast requirements for
informative graphical elements.~~
![image](https://github.com/owncloud/ios-app/assets/132444655/cd1c991c-f9dc-4310-9521-f77154aec3f7)

**-> The colors for the brighter text and the icons are using standard system-provided colors used by iOS throughout the system. Enabling the option `Increase Contrast` (`Kontrast erhöhen`) in System Settings also increases the contrast here:**

Without `Increase Contrast` | With `Increase Contrast`
---|----
![Image](https://github.com/user-attachments/assets/4be989ad-5fe7-4ae4-a84a-fa41b95b5c9d) | ![Image](https://github.com/user-attachments/assets/8351a218-4bb7-4c65-9dcc-3ff5425a5d97)

- ~~[ ] The frames of the checkboxes do not meet the contrast requirements for graphic
contrast requirements for graphical control elements.~~

 **-> The checkboxes used here are system-provided UI elements used throughout iOS. Enabling the option `Increase Contrast` (`Kontrast erhöhen`) in System Settings also increases the contrast here:**

Without `Increase Contrast` | With `Increase Contrast`
---|----
![Image](https://github.com/user-attachments/assets/d1a4ddac-ccce-4098-a3d4-9f3be6fab7f9) | ![Image](https://github.com/user-attachments/assets/e04aa59c-488a-46f6-9eab-2b2a6f176ff2)

**Here's the option to enable in System Settings (at Einstellungen > Bedienhilfen > Anzeige & Textgröße):**
![Image](https://github.com/user-attachments/assets/9e1ab512-7c51-46db-8561-3956d5eafccb)

**Comment by jesmrec** [comment]
Let's dive a little bit

|| Regular color | High contrast color |
| :-: | :-: | :-: |
| **Row badges** | #8a8a83  (3,47) ✅  | #838386 (3,78) ✅ |
| **Multiselector** | #c5c5c7 (1,72) ❌  | #77777b (4,46) ✅  |

(color values are approximated)

That means, contrast requirements are fulfilled (up to 3:1) in three out of four cases. Multiselector regular color does not meet the requirements, but, using the high contrast option the problem is fixed.

**Comment by felix-schwarz** [comment]
Thanks for verifying! Since the issue is fixed when enabling High Contrast in the settings and users in need of a high contrast are highly likely to enable it, I consider this issue addressed and close the issue.
